### ISSUE TITLE
Dutch quotation marks changed from low-high double to high-high double too quickly. At least discuss this. Without a discussion, revert or provide the user with a switch, the low-high double being the default.

### ISSUE THREAD

**Issue by AlMa1r** [josephwright/csquotes#71]
A discussion from several (at least more than one) experts in Dutch and native Dutch speakers is missing in https://github.com/josephwright/csquotes/pull/64 ; the change occurred on a vote of only one user, whose level of expertise in this matter is not (or not clearly) visible.
According to [Wikipedia](https://en.wikipedia.org/wiki/Quotation_mark) as of 2024-04-10, the primary form is `„”`, and «The standard form in the preceding table is taught in schools and used in handwriting. Most large newspapers have kept these low-high quotation marks, `„` and `”`; otherwise, the alternative form with single or double English-style quotes is now often the only form seen in printed matter.».
The lack of clarity in «often the only» put aside, if the part before the semicolon is true, then the low-high double quotation marks is the school and not a “very old-school” way, and certain credible media have been typesetting the low-high double form for a while (perhaps, ever since). Moreover, recompiling old LaTeX documents now leads to different results for no good reason :-(. 
In the past, the high-high double quotation marks could have been favoured because of the keyboards of typewriters and computers and ASCII. However, the significance of this argument has decreased since the advent of a wide range of input methods and of character encodings. For instance, with my keyboard and my language settings in Linux, I have been routinely typesetting `„` using AltGr+V, `“` using AltGr+B, and `”` using AltGr+N for years. Some folks use screen keyboards, projections of keyboards onto the table surface, physical keyboards with letters displayed by mini-screens (e.g., Optimus Maximus), … meanwhile. In such cases (potentially, after tuning), the effort for the two forms is the same.
So though the reasons for a change are not completely invalid (cf. https://op.europa.eu/en/web/eu-vocabularies/formex/physical-specifications/character-encoding/use-of-quotation-marks-in-the-different-languages and https://onzetaal.nl/taalloket/aanhalingstekens-hoog-of-laag), they are weak.  If the US keyboard layout and ASCII turn out to be the only real reason for using the high-high quotation marks, then it does not constitute an improvement. In intentionally strong terms, which might hurt a reader's feelings, it is corruption.
Therefore, unless a proper discussion takes place and new and better arguments emerge, it'd be best to stick to the low-high (old, standard, school) double form as default and provide the user with a switch for the alternative high-high double form.

**Comment by jasperhabicht** [comment]
To my understanding, there is no official rule, but the traditional form is `„”` (not `„“`) and the modern form `“”`. 

I am voting for an additional style (such as in Danish with `quotes` and `topquotes`) which allows for switching between these two styles since the traditional style seems to be in use as well as the modern.

**Comment by AlMa1r** [comment]
@jasperhabicht By the way, the term “modern” sounds positive, perhaps an improvement, whereas we might wish to consider the issue without any emotions. Perhaps, _high-high_ and _low-high_ could be good terms for this purpose.
Thanks for the correction of `„“` to `„”`! I've improved this in my post.

**Comment by josephwright** [comment]
The PR in question has a link to what seems to me to be an official language body - which is the sort of thing I look for in accepting a change (also noting that the original version was marked as 'not sure' by PL).

**Comment by AlMa1r** [comment]
> The PR in question has a link to what seems to me to be an official language body - which is the sort of thing I look for in accepting a change (also noting that the original version was marked as 'not sure' by PL).

No—and feel free to correct me if I'm wrong—I don't think that Onze Taal is a strictly “*official* language body”, at least not as strong as Rechtschreibrat for German or Académie Française for French.

Still, yes, I do acknowledge their contribution fully—no argument here, and their argument is still both valid and weak because of “[…] in the course of the twentieth century under the influence of technology. Because keyboards […]”. The US technology corrupted the Dutch language. The majority (here: the folks who used typewriters and ASCII on PCs and did a quick-and-dirty typesetting in the 20th century) is sometimes wrong.

It seems to me that Onze Taal is not prescriptive—and again feel free to correct me if I'm wrong—i.e., it's more like Duden-Redaktion than like Rechtschreibrat or Académie Française. IMHO, Onze Taal tries to be all-inclusive and big rather than pure, precise, and accurate (to compare to German, it seems to be more like Duden with certain book publishers, who accept everything, than like Wahrig with the news agencies, who try to be more precise and make fine-grained distinctions whenever there is a choice).

As of now, Onze Taal says it has become “a general platform for language lovers” and a “cultural public benefit institution” and no longer an association of purists. But even if Onze Taal were pure and prescriptive, we still have their word that, “some newspapers and magazines still use them”, i.e., the low-high quotation marks. So such publishers and the authors publishing there have to be accounted for.

In addition, [the main Wikipedia article in Dutch on the quotation marks](https://nl.wikipedia.org/wiki/Aanhalingsteken) as of 2024-04-11 also uses the low-high double form in the very first examples of the quoted space, a quoted letter, and a quoted sentence:

> „ ”
> „O”
> Jan zei: „Hallo.”

I believe that if the high-high double opposition including Onze Taal were really strong and cared, they would have changed the first examples long ago.

And even if I were hypothetically wrong on every single point above, we still don't see a discussion of the native-speaker experts in Dutch to advocate for the change.

**Comment by jasperhabicht** [comment]
> I don't think that Onze Taal is a strictly “official language body”, at least not as strong as Rechtschreibrat for German or Académie Française for French.

I am actually not sure that there is any official rule that specifies *which* kind of quotation marks are the correct ones for German (set aside the variants for Swiss German). The examples used in several official documents regulating the German language *use* low-high marks, but I did not find any specific rule. Prove me wrong though. But against this background, I am unsure whether one can argue that Onze Taal is not reliable in this regard.

**Comment by AlMa1r** [comment]
> I am actually not sure that there is any official rule that specifies _which_ kind of quotation marks are the correct ones for German (set aside the variants for Swiss German). The examples used in several official documents regulating the German language _use_ low-high marks, but I did not find any specific rule. Prove me wrong though. But against this background, I am unsure whether one can argue that Onze Taal is not reliable in this regard.

Strictly speaking, I have not called Onze Taal unreliable in this regard. (When I think of it further, it's unclear on what “reliable” would really mean in this context—let me not expand on this.) At the same time, I find their argument weak, and I find the opinion of other typesetters for Dutch important, too, and, as you rightfully mentioned in the TeX.SE chat, a proper discussion by a wider community (I'm thinking of the relevant folks: native Dutch speakers and experts in Dutch) should have taken place prior to any so significant changes.

(Off-topic: As for German, cf. DIN 5008-2011, § 6.3 Anführungszeichen. The norm shows which kinds are to be used in proportional and typewriter fonts by examples and without any explanation of the forms of those marks. Unfortunately, I don't have the update DIN 5008-2020; if you have access to it, please feel free to quote from there. At least one [guideline to DIN 5008-2020](https://www.uni-regensburg.de/assets/zentrum-sprache-kommunikation/dokumente/schreibberatung/Leporello-DIN5008_Linksbund.pdf) does mention the 99 and 66 forms. Anyway, the usage of the quotation marks in German is incomparable to their usage in Dutch because there's no controversy in German.)

It has just occurred to me that I have not typeset any single Dutch quotation mark myself. Biblatex (using csquotes) did it for me in

> Edsger Wybe Dijkstra. “EWD74. Over seinpalen”. Privaat gecirculeerd. Eindhoven, Netherland, 1964 or 1965.

a few days ago and

> Edsger Wybe Dijkstra. „EWD74. Over seinpalen”. Privaat gecirculeerd. Eindhoven, Netherland, 1964 or 1965.

a month ago. In the 1980s and early 1990s, before the spread of TrueType and Unicode, I agree, I would have probably gone for the high-high double version myself on a typewriter or in an editor on CP/M or MSDOS. But now that's pointless.

**Comment by josephwright** [comment]
> I find the opinion of other typesetters for Dutch important, too, and, as you rightfully mentioned in the TeX.SE chat, a proper discussion by a wider community (I'm thinking of the relevant folks: native Dutch speakers and experts in Dutch) should have taken place prior to any so significant changes.

As a package author, the issue for me is that realistically one never gets much in the way of wider engagement in any discussion. Native _speakers_ are not useful here: one needs experts in the typographic traditions, and those people are rare and not necessarily easy to find. Moreover, gaining a consensus is far from certain. At the same time, it's hard to judge the true status of 'official' groups - even if they have some formal status, doesn't mean that they are listened to! So in the end you tend to do your best based on limited information.

**Comment by AlMa1r** [comment]
> > I find the opinion of other typesetters for Dutch important, too, and, as you rightfully mentioned in the TeX.SE chat, a proper discussion by a wider community (I'm thinking of the relevant folks: native Dutch speakers and experts in Dutch) should have taken place prior to any so significant changes.
> 
> As a package author, the issue for me is that realistically one never gets much in the way of wider engagement in any discussion. Native _speakers_ are not useful here: one needs experts in the typographic traditions, and those people are rare and not necessarily easy to find. Moreover, gaining a consensus is far from certain. At the same time, it's hard to judge the true status of 'official' groups - even if they have some formal status, doesn't mean that they are listened to! So in the end you tend to do your best based on limited information.

If I had time, I'd open a few posts (on TeX.SE, topanswers.xyz, latex.org, … ) to attract attention to this issue. There are definitely thousands of LaTeX users knowledgeable in Dutch typography, only they are not yet aware of the issue.

Now I looked onto the Web sites of the main national newspapers according to https://en.wikipedia.org/wiki/List_of_newspapers_in_the_Netherlands. The first three mostly widespread (*De Telegraaf*, *Algemeen Dagblad*, *de Volkskrant*) use the high-high single quotation marks `’ ’` or `‘ ’`, whereas the fourth newspaper *NRC* indeed uses the low-high double quotation marks:

> Verder: De door Iran gesteunde militante Libanese beweging Hezbollah zegt vrijdag „tientallen raketten” te hebben afgevuurd op Israëlische doelen als vergelding voor Israëlische aanvallen op woningen en dorpen in het zuiden van Libanon.

(Source: https://www.nrc.nl as of 2024-04-13.)

Only the seventh newspaper from the top, *Reformatorisch Dagblad*, uses the high-high double quotation marks, but they don't pay attention:

> Napalmmeisje Kim Phúc: „Vroeger vond ik het onbegrijpelijk dat die foto gemaakt is, nu ben ik er dankbaar voor”
> ”Gouden geluk” betekent haar naam.

(Source: https://www.rd.nl as of 2024-04-13.)

So if we take the order from this empirical observation (which, of course, represents partial knowledge only), then the pair of the low-high double quotation marks still beats the pair of the high-high double quotation marks. IMHO, yet another reason to revert the change and/or provide the user with a choice.

(Off-topic: I guess, someone could try to make a case for the single ones, but that's not what was advocated for in https://github.com/josephwright/csquotes/pull/64 .  Further, I suspect that many Dutch newspapers have chosen the single ones to save space, similarly to why certain British newspapers dropped the serial comma long, long ago.)

**Comment by AlMa1r** [comment]
> experts in the typographic traditions

Asked for in https://typography.guru/forums/topic/139728-experts-in-dutch-typographic-traditions-are-asked-about-the-proper-quotation-marks-for-csquotes-in-latex .

**Comment by pereelmagne** [comment]
I just looked at Unicode’s CLDR standard for Dutch and there the system is “” (high-high).
https://www.unicode.org/cldr/charts/45/summary/nl.html

**Comment by AlMa1r** [comment]
> I just looked at Unicode’s CLDR standard for Dutch and there the system is “” (high-high). https://www.unicode.org/cldr/charts/45/summary/nl.html

Thank you! No: If you refer to lines 28 and 29, where the cells in the columns _Header_ and _Code_ contain _Quotation marks_ and _Start_/_End_, then we see the high-high double `”`/`“` in the column _English_, whereas the column _Native_ has high-high single quotation marks. The way I read it, the Unicode chose the 9/6 high-high single form `’‘` for Dutch. Cf. the aside at the end of https://github.com/josephwright/csquotes/issues/71#issuecomment-2053258105 .

**Comment by pereelmagne** [comment]
Unfortunately, I don’t have insight in Unicode’s debates. My take on this is: 
- Editing conventions are rarely regulated (international symbols, some uses of capitals…).
- They are born and grow and ultimately go away in real texts, in a long process, difficult to trace (tradition).
- Wikipedia and Unicode can serve as points of reference to state that at present the model is high-high, but the use of low-high is still valid in texts such as essays, etcetera. (But I’m not familiar with the situation in the Netherlands, so please feel free to ignore me.)

Moreover, when I watch Japanese or Korean series, I see that people write horizontally and left-to-right, even though traditionally those scripts go right-to-left and vertical. That means that there is some kind of globalization phenomenon in editing conventions.

**Comment by AlMa1r** [comment]
Thank you!

>     * Wikipedia […] can serve as [point] of reference to state that at present the model is high-high

No: at least as of 2024-04-24, https://en.wikipedia.org/wiki/Quotation_mark and https://nl.wikipedia.org/wiki/Aanhalingsteken provide us with the low-high double `„”` as the model for Dutch.

**Comment by AlMa1r** [comment]
Asked in http://tex.stackexchange.com/questions/720953 .

**Comment by ilayn** [comment]
Dutch traditional model is low-high but that's really going back a lot. The current quotes are high-high and users don't freak out when they see it, and that should probably settle the issue. I don't think anyone expects low-high in text these days but single quotes. But often, it is used as an ornament in the newspapers similar to how ĳ is playfully used.

Here is a nice post about a lot of typographical intricacies of quotation marks that I enjoy reading https://type.today/en/journal/quotes There is list at the bottom for various languages with secondary options for nesting.

By the way; Joseph is there any package you are not maintaining now 😅 ?

**Comment by renkema** [comment]
> Native speakers are not useful here: one needs experts in the typographic traditions, and those people are rare and not necessarily easy to find. Moreover, gaining a consensus is far from certain. At the same time, it's hard to judge the true status of 'official' groups

Onze Taal is an influential group – popular enough to [compete with the official spelling body](https://en.wikipedia.org/wiki/Witte_Boekje) – and in general a dependable source on orthographical matters. It seems to me, though, that their position was misrepresented in issue #64. The [article linked to there](https://onzetaal.nl/taalloket/aanhalingstekens-en-leestekens) does not even concern quotation marks, but the [article mentioned by AlMa1r](https://onzetaal.nl/taalloket/aanhalingstekens-hoog-of-laag) does, and makes the following points:

- There is no correct or official way of writing quotation marks [indeed, the style varies between publishers and genres; browsing my own bookshelves, high `“”` quotes seem correlated with poor line breaking].
- Double low-high quotes (`„”`) are traditional [hence the NRC using them to signal prestige, and the RD using them to signal conservatism].
- The recent popularity of high quotes (`“”`) is likely a result of limited functionality of computer software.
- [From [linked](https://onzetaal.nl/taalloket/enkele-aanhalingstekens) [posts](https://onzetaal.nl/taalloket/aanhalingstekens-dubbele-aanhalingstekens):] By a common convention, double quotes (whether `„”` or `“”`) are used for direct speech only and high single quotes (`‘’`) are used for all other cases.

I added the last point because this distinction is apparently not commonly made in the Anglophone world (where quotation marks change for nested quotations instead – there is no such convention in Dutch writing). It much more common in Dutch, however, and the source of confusion upthread on the RD style.

From all this, the conclusion I would draw is that high-end typesetting software should have the technically-complex-yet-traditional option as the default. After all, was breaking free of the constraints of DTP software not one of the core principles of (La)TeX? And given the great variation in quotation styles, it seems users with opinions of their own will want to set their preference explicitly anyway.

**Comment by AlMa1r** [comment]
@renkema 
> confusion upthread on the RD style

Thanks on making us aware of that. So is

> Napalmmeisje Kim Phúc: „Vroeger vond ik het onbegrijpelijk dat die foto gemaakt is, nu ben ik er dankbaar voor”
”Gouden geluk” betekent haar naam.

(source: *Reformatorisch Dagblad*, https://www.rd.nl on 2024-04-13)  fine?

**Comment by renkema** [comment]
Yes, that would be fine: the low quotes mark direct speech and the high quotes mark a literal translation.

**Comment by archifont** [comment]
@josephwright, would it be possible implement a choice and to add `dutch` as a language option in Table 1, similar to how you included `guillemets` and `quotes`, and to incorporate the values `quotes` (`“”` top quotes) and `oldstylequotes` (or another value name that shows a difference) (`„”` bottom opening quote and top closing quote)?

I have no preference regarding the default setting, but I would appreciate having the option, as both styles are still quite commonly used in the Netherlands.

**Comment by AlMa1r** [comment]
> [@josephwright](https://github.com/josephwright), would it be possible implement a choice and to add `dutch` as a language option in Table 1, similar to how you included `guillemets` and `quotes`, and to incorporate the values `quotes` (`“”` top quotes) and `oldstylequotes` (or another value name that shows a difference) (`„”` bottom opening quote and top closing quote)?
> 
> I have no preference regarding the default setting, but I would appreciate having the option, as both styles are still quite commonly used in the Netherlands.

Isn't it the case that all four kinds are used in the Netherlands (and that the publishers, including newspapers, have a great deal of variation in their preferences)? Moreover, we'd probably wish to make the option values neutral and avoid saying “old” or “new”. Suggestion: `lohisingle`, `hihisingle`, `lohidouble`, and `hihidouble`. Or `lowtopsingle`, `toptopsingle`, `lowtopdouble`, and `toptopdouble`. Also https://github.com/josephwright/csquotes/issues/71#issuecomment-2186385034 is important, but I don't know how to automate this.

**Comment by archifont** [comment]
> > [@josephwright](https://github.com/josephwright), would it be possible implement a choice and to add `dutch` as a language option in Table 1, similar to how you included `guillemets` and `quotes`, and to incorporate the values `quotes` (`“”` top quotes) and `oldstylequotes` (or another value name that shows a difference) (`„”` bottom opening quote and top closing quote)?
> > I have no preference regarding the default setting, but I would appreciate having the option, as both styles are still quite commonly used in the Netherlands.
> 
> Isn't it the case that all four kinds are used in the Netherlands (and that the publishers, including newspapers, have a great deal of variation in their preferences)? Moreover, we'd probably wish to make the option values neutral and avoid saying “old” or “new”. Suggestion: `lohisingle`, `hihisingle`, `lohidouble`, and `hihidouble`. Or `lowtopsingle`, `toptopsingle`, `lowtopdouble`, and `toptopdouble`. Also [#71 (comment)](https://github.com/josephwright/csquotes/issues/71#issuecomment-2186385034) is important, but I don't know how to automate this.

You are correct that all four types of quotation marks are used today through publishers’ style guides containing their specific preferences. Regarding the old and new rules, I was referring to the built-in spell checker in Adobe software—specifically InDesign—which gives two options for Dutch: `Dutch: Old Rules` and `Dutch: 2005 Reform`. However, this pertains to spelling, and the preferences for quotation marks did not change in 2005. Therefore, if this request gets implemented, more neutral value options would be better.

Considering automating this with some (`Lua`)(`La`)`TeX` logic would indeed be helpful, as an intermediate step. I don't know either how to do that though.

**Comment by AlMa1r** [comment]
I revise my prior suggestion a tiny bit. The antonyms of “top” are more “bottom” and “base”, as both can act as nouns, and not “low”, which is an adjective. So the options names should read

1. `lohisingle`, `hihisingle`, `lohidouble`, and `hihidouble`, or
2. `bottomtopsingle`, `toptopsingle`, `bottomtopdouble`, and `toptopdouble`, or
3. `basetopsingle`, `toptopsingle`, `basetopdouble`, and `toptopdouble`.

Items 2 and 3 could be abbreviated: `btsingle`, `ttsingle`, `btouble`, and `ttdouble`. (Caveat: this might make folks unhappy who see `tt` only as an abbreviation for “teletype”. You can't make everyone happy.)
